tests/test_processing_reaper.py

```
@@ -0,0 +1,245 @@
+    svc = QueueService()
+    svc.queue_key = "test:email:queue"
+    svc.processing_key = "test:email:processing"
+    svc.processing_visibility_key = "test:email:processing:vis"
+    svc.delayed_queue_key = "test:email:delayed"
+    svc.dlq_key = "test:email:dlq"
+    svc.queued_set_key = "test:email:queued:set"
+
+    client = await redis.from_url("redis://localhost:6379/0", decode_responses=True)
+    try:
+        await client.ping()
+    except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)